Domain.Tests/CommandSchedulingTests.cs

```diff
@@ -480,12 +481,14 @@ await configuration.CommandScheduler<Order>()
 
             // initialize twice
             new AnonymousCommandSchedulerPipelineInitializer(cmd => commandsScheduled.Add(cmd))
-                .Initialize(configuration);
+                .Initialize(Configuration.Current);
+
             new AnonymousCommandSchedulerPipelineInitializer(cmd => commandsScheduled.Add(cmd))
-                .Initialize(configuration);
+                .Initialize(Configuration.Current);
 

```

Comment:
We shouldn't do this. There's a global `Configuration` object which ideally is never used in tests, since it can result in sharing state between tests.